applications/serial_lte_modem/prj.conf

```diff
@@ -80,11 +80,10 @@ CONFIG_FCB=y
 #
 # SLM-specific configurations
 #
+CONFIG_SLM_CUSTOMIZED=n
 CONFIG_SLM_LOG_LEVEL_INF=y
-# Configure external XTAL for UART
 CONFIG_SLM_EXTERNAL_XTAL=n
-# Start up then enter sleep
-#CONFIG_SLM_START_SLEEP=y
+CONFIG_SLM_START_SLEEP=n
```

Comment:
Are changes in this file related somehow?

Reply:
No, not related. Some reviewer asked me to remove unnecessary comments from another prj.conf so I did the same here, too